db/migrate/20190118145417_create_votes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateVotes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :votes do |t|
+      t.timestamps
```

Comment:
Missed these earlier, I think all 3 columns should not be nullable, right? Except maybe `user`?

Reply:
Change made now, timestamps have it generated automatically and user we want to be able to null so if a user is removed their vote stays on an idea.  So just idea_id that should not be null.